tutorials/tutorial-01.qmd

```diff
@@ -6,33 +6,1403 @@ time: "~45 minutes"
+
+- Explore and discover available [WFS](../concepts.qmd#wfs-web-feature-service) layers using `emodnet.wfs` functions
+- Retrieve multiple vector datasets representing MPAs and subsea infrastructure
+- Harmonise [coordinate reference systems](../concepts.qmd#coordinate-reference-systems-crs) across layers
+- Apply spatial intersection techniques to detect overlaps
+- Summarise and visualise the extent of intersection between protected areas and human activities
 
 ### Data Sources
 
-**Protected Areas (Seabed Habitats):**
-- `cddaareas`
-- `natura2000areas`
-- `marineprotectedareas`
+All data comes from the **EMODnet Human Activities** portal:
+
+**Protected Areas:**
+
+| Layer | Description |
+|-------|-------------|
+| `cddaareas` | CDDA (Common Database on Designated Areas), the European inventory of nationally designated protected areas including nature reserves, national parks, and other site types |
+| `natura2000areas` | Natura 2000 network, the EU-wide network of protected areas established under the Birds and Habitats Directives to protect Europe's most valuable species and habitats |
+| `marineprotectedareas` | Other marine protected areas, sites designated under regional sea conventions (OSPAR, HELCOM, Barcelona Convention) and other international frameworks |
+
+**Energy Infrastructure:**
+
+| Layer | Description |
+|-------|-------------|
+| `platforms` | Offshore platforms, including oil, gas, and wind energy installations such as production platforms, substations, and support structures |
+| `pipelines` | Subsea pipelines for oil, gas, and chemical transport on the seabed |
+| `pcablesbshcontis` | Power cables from BSH (German Federal Maritime and Hydrographic Agency) and CONTIS database |
+| `pcablesshom` | Power cables from SHOM (French Naval Hydrographic and Oceanographic Service) |
+| `pcablesrijks` | Power cables from Rijkswaterstaat (Dutch Directorate-General for Public Works and Water Management) |
+| `pcablesnve` | Power cables from Norwegian sources |
+
+Note: EMODnet also provides telecommunication cable datasets (e.g., `sigcables`, `ukfibrecables`), but we focus on energy infrastructure in this tutorial.
+
+## Setup
+
+### Packages
+
+We'll use `emodnet.wfs` to access EMODnet WFS services, `sf` for spatial data handling, `dplyr` and `purrr` for data manipulation, and `ggplot2` and `tmap` for visualisation.
+
+```{r}
+#| label: load-packages
+#| message: false
+library(emodnet.wfs)
+library(sf)
+library(dplyr)
+library(purrr)
+library(ggplot2)
+library(tmap)
+```
+
+```{r}
+#| label: setup
+#| include: false
+# Load package data (mpa_attr_tbls, infra_attr_tbls) and helper functions
+library(emodnet.bio.r.geo.tutorials)
+```
+
+::: {.callout-tip collapse="true"}
+## Package installation
+If you haven't installed these packages, run:
+
+```r
+install.packages(c("emodnet.wfs", "sf", "dplyr", "purrr", "ggplot2", "tmap"))
+```
+
+For the latest development version:
+
+```r
+# install.packages("pak")
```

Comment:
```suggestion
```

Reply:
Leaving this as GitHub install since it's specifically for the development version. We should discuss the r-universe release strategy separately. My vision was to only publish released versions there to avoid dev bugs being installed by users. For emodnet.wfs, r-universe would just act as a mirror, but for emodnet.wcs it allows us to have a stable binary version and act a bit like CRAN until it's on there too.